docker.dev.yml

```diff
@@ -1,10 +1,22 @@
 version: "3"
 
 services:
-  app:
-    container_name: scribble-ocr
+  train:
+    container_name: scribble-ocr-train
     build:
       context: .
+      target: tesseractbuild
+      dockerfile: Dockerfile_tesseract_training
+    ports:
+      - 8000:8000
+    volumes:
+      - "./data:/train/tesstrain/data"
+    command: sh /app/start.sh
+  api:
```

Comment:
The cmd of this container doesn't really make sense because when running the api we really want to run the uvicorn stuff that was in `Dockerfile`. The start.sh script doesn't do anything useful for the API